frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/cover/cover_node_widget.dart

```diff
@@ -435,8 +435,18 @@ class _CoverImageState extends State<_CoverImage> {
     final Widget coverImage;
     switch (selectionType) {
       case CoverSelectionType.file:
+        final imageFile =
+            File(widget.node.attributes[kCoverSelectionAttribute]);
+        if (!imageFile.existsSync()) {
+          // reset cover state
+          WidgetsBinding.instance.addPostFrameCallback((_) {
+            widget.onCoverChanged(CoverSelectionType.initial, null);
+          });
+          coverImage = const SizedBox();
+          break;
+        }
```

Comment:
@LucasXu0.  Here I added backup logic if file was not found reset to no cover state

Reply:
Got it.